lib/hanami/assets/helpers.rb

```diff
@@ -682,6 +752,15 @@ def _typed_asset_path(source, ext)
         asset_path(source)
       end
 
+      def _sri?
+        !!self.class.assets_configuration.sri
+      end
+
+      def _sri_value(source, ext)
+        source = "#{ source }#{ ext }" unless source.match(/#{ Regexp.escape(ext) }\z/)
+        self.class.assets_configuration.sri_value(source) unless _absolute_url?(source)
```

Comment:
@accuser I'd stick with HTML attributes:

Return the value from the manifest:

```erb
<%= javascript 'application', integrity: true %>
```

Disable for this tag:

```erb
<%= javascript 'application', integrity: false %>
```

Force a different value:

```erb
<%= javascript 'application', integrity: 'sha512-abcdef' %>
```

Does it makes sense?